matrixprofile/algorithms/cympx.pyx

```diff
@@ -78,7 +78,7 @@ cpdef mpx_parallel(double[::1] ts, int w, bint cross_correlation=0, int n_jobs=1
     cross_cov(cov, ts[minlag:], mu[minlag:], first_seq)
     
     # this is where we compute the diagonals and later the matrix profile
-    diff_equations(df, dg, ts, mu, w)
+    diff_eqns(df, dg, ts, mu, w)
```

Comment:
Any reason why the function name is abbreviated? It is preferred to be verbose.

Reply:
If I recall correctly, it was an issue of line width somewhere. Also, I didn't know verbose was preferred. I can roll that back.

Reply:
Go ahead and keep it spelled out. This aligns with PEP standards.